WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -759,7 +759,6 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             }
             if (mViewPager.getCurrentItem() > PAGE_CONTENT) {
                 if (mViewPager.getCurrentItem() == PAGE_SETTINGS) {
-                    mPost.setFeaturedImageId(mEditPostSettingsFragment.getFeaturedImageId());
```

Comment:
Same situation as my other comment, it's making sure that any changes to the featured image in the post settings is reflected in the editor. Featured image is the only 'post setting' that can be changed from the editor, which is why it's had this special treatment.